samples/unit-tests/rangeselector/legend-position/demo.js

```diff
@@ -0,0 +1,24 @@
+QUnit.test('Legend should be rendered below the range selector', function (assert) {
+    Highcharts.stockChart('container', {
+        rangeSelector: {
+            enabled: true,
+            inputEnabled: false,
+            buttonPosition: {
+                align: 'right'
+            }
+        },
+        legend: {
+            enabled: true,
+            align: 'left',
+            verticalAlign: 'top'
+        },
+        series: [{
+            data: [101, 343, 11]
+        }]
+    });
+    assert.ok(
+        document.getElementsByClassName('highcharts-legend')[0].getBoundingClientRect().top >=
```

Comment:
Don't use `document` - you may get elements from test-templates that are existing in the background. Use `chart.container` instead.